Projects/DEV/DEV03/DEV03-AWS100.md

```diff
@@ -0,0 +1,40 @@
+# DEV03-AWS100 —  Install & Configure AWS CLI then create an S3 Bucket
+
+## Cloud Service Provider
+* Amazon Web Services (AWS)
+
+## Difficulty
+Level 100 (Introductory)
+
+## Estimated Time
+* 30 - 40 minutes
+  
+## Project's Author(s)
+* [Ariela](https://twitter.com/ari_hacks)
+
+## Objectives
+
+###  You need to complete the following:
+
+* Create an IAM user (with Administrator Access) if you do not have one already
+* Install AWS CLI 
+* Configure AWS credentials locally: `aws configure` 
+* Create an S3 bucket: `aws s3 mb s3://<unique-bucket-name>`
+* Check the bucket was created: `aws s3 ls`
+* Delete the bucket when you are done: `aws s3 rb s3://<unique-bucket-name>`
+
+###  You need to answer the following: 
+
+* How are permissions granted to IAM users?
+* What settings are created when configuring AWS locally and where are they stored? 
```

Comment:
`settings` should be `files`

Reply:
I was referring to the settings stored in the files, as mentioned in the [docs](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html). Maybe it would be more clear if I said 'What credentials'?

Reply:
@ari-hacks Yes, I think that would clarify it :)